### DISCUSSION TITLE
Im Confused plz help

### DISCUSSION THREAD

**Discussion by brxndonnn** [new92/igfi#23] [Q&A] [answered]
what do i do with "python3 mainV and the name of the folder. Example: python3 mainV1.py.py"

oh and how do i run it lol

**Comment by new92** [comment, marked as answer]
Hey there ! Ok so this command is used to actually execute the script from the terminal. But it's just optional. If you wish you can execute the script also from the IDE you're using. And the usage of this command is: python <and you enter here the name of the script>.py .Hope this helped

**Reply by brxndonnn** [reply]
Its working now, Thank you very much for the response!

**Comment by SND31** [comment]
Me podrías explicar paso a paso ejecutar correctamente esta app ya la tengo instalada pero de hay a hacerla funcionar no me doy una buena idea de cómo va todo
